.github/workflows/CI-python.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       matrix:
         packageDirectory: ["ml_wrappers"]
         operatingSystem: [ubuntu-latest, macos-latest, windows-latest]
-        pythonVersion: [3.6, 3.7, 3.8, 3.9]
+        pythonVersion: [3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
nit: should all of them be quoted then?  or maybe remove the quotes on 3.10?

Reply:
I think having it as 3.10 makes the workflow detect it as 3.1 that's why for 3.10 we need to have it as '3.10'.